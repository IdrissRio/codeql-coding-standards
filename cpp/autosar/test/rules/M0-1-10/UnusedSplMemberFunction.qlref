rules/M0-1-10/UnusedSplMemberFunction.ql