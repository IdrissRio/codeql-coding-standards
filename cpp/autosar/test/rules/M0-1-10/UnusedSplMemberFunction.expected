| test.cpp:71:5:71:16 | ANestedClass | Special member function ANestedClass is never called. |
| test.cpp:82:5:82:22 | AnotherNestedClass | Special member function AnotherNestedClass is never called from a main function or entry point. |
